application/wlp-web-java-spring/src/main/java/webapp/tier/util/HazelcastInstanceConfigurator.java

```diff
@@ -7,13 +7,18 @@
 import com.hazelcast.client.config.XmlClientConfigBuilder;
 import com.hazelcast.core.HazelcastInstance;
 
-import webapp.tier.util.GetConfig;
+public class HazelcastInstanceConfigurator {
```

Comment:
Codacy found an issue: [The utility class name 'HazelcastInstanceConfigurator' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=6170659)